packages/cli/src/lib/live-server/index.js

```diff
@@ -421,8 +421,22 @@ LiveServer.start = function(options) {
     }
 
     // Only reload active tabs if the changed file is opened in one of them
+    const handleHtmlUrl = (url) => {
+      if (!url || url.length === 0) {
+        return "index.html";
+      }
+
+      if (url.endsWith('.html')) {
+        return url;
+      }
+
+      return url.replace(/\/$/g, '').concat('/index.html');
+    };
+
     let normalizedPath = fsUtil.ensurePosix(path.relative(root, changePath));
-    if (LiveServer.activeTabs.some(tab => tab.client && tab.url === normalizedPath)) {
+    if (LiveServer.activeTabs.some(tab => {
+      return tab.client && handleHtmlUrl(tab.url) === handleHtmlUrl(normalizedPath);
+    })) {
```

Comment:
I think only the call for `tab.url` is necessary, as that is the only variable that is possible to have the `*/` form. `normalizedPath` refers to the normalized path of an existing file that was just changed, so it most likely never holds the `*/` form, and therefore the call on it would be doing nothing.

I'd opt to use implicit return instead as it is just a short single return statement.

Reply:
Thanks for the suggestions, fixed all the pointed issues.